pkgs/development/python-modules/keystoneauth-websso/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  keystoneauth1,
+  multipart,
+  pythonOlder,
+  setuptools,
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "keystoneauth_websso";
+  version = "0.2.0";
+
+  disabled = pythonOlder "3.8";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-84GtBYOh/kYmMMk5Voyflw3NrBuxkV/Vj+ih8SbwzAo=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeCheckInputs = [ poetry-core ];
+
+  propagatedBuildInputs = [
+    keystoneauth1
+    multipart
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+    runHook postCheck
+  '';
+
+  pythonImportsCheck = [ "keystoneauth_websso" ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increases eval times

---

done.